packages/services-extension/src/index.ts

```diff
@@ -141,14 +162,14 @@ const kernelSpecManagerPlugin: ServiceManagerPlugin<KernelSpec.IManager> = {
   description: 'The kernel spec manager plugin.',
   autoStart: true,
   provides: IKernelSpecManager,
-  requires: [IKernelSpecs],
+  requires: [IKernelSpecClient],
   optional: [IServerSettings],
   activate: (
     _: null,
-    kernelSpecs: IKernelSpecs,
+    kernelSpecAPIClient: IKernelSpecClient,
     serverSettings: ServerConnection.ISettings | undefined,
   ): KernelSpec.IManager => {
-    return new LiteKernelSpecs({ kernelSpecs, serverSettings });
+    return new KernelSpecManager({ kernelSpecAPIClient, serverSettings });
```

Comment:
FYI @martinRenou this defaults back to the JupyterLab default `KernelSpecManager`, and instead JupyterLite provides a smaller and simpler `IKernelSpecClient`. And somehow undoes https://github.com/jupyterlite/jupyterlite/pull/1596.

This should not have any effect on `jupyterlite-xeus` though, since the signal should be correctly emitted in the upstream implements of the `KernelSpecManager`.